components/project-controller/development/kustomization.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- ../base
+- https://github.com/konflux-ci/project-controller/config/default?ref=9937d7d5caf9625823889a8ba6865bcbd4343ad8
+
+images:
+- name: konflux-project-controller
+  newName: konflux-project-controller
```

Comment:
If you set the `newName` to only `konflux-project-controller`, the resulting image to be pulled as part of deployment is image: `konflux-project-controller:9937d7d5caf9625823889a8ba6865bcbd4343ad8` Is this intentional, i.e. have a image name that is not fully qualified like `quay.io/...`

Reply:
As @ifireball mentioned, the name shouldn't be redhat specific. So I removed qua.io/redhat-appstudio.

Reply:
IIUC, the name is not redhat specific in project-controller repo but here in infra-deployments, we transform the image to be pulled from somewhere and that place is quay, which is fine

Reply:
Ok, will update the `newName`.